py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -183,6 +183,31 @@ def aten_ops_native_group_norm(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.gather.default)
+@enforce_tensor_types(
+    {
+        0: (TRTTensor,),
+    }
+)  # type: ignore[misc]
+def aten_ops_gather(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.select.gather(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        input=args[0],
+        dim=args[1],
+        index=args[2],
+        sparse_grad=args_bounds_check(args, 3, False),
```

Comment:
Based on the schema [here](https://github.com/pytorch/pytorch/blob/4cdc2d7096ee030b35b666a70d48aeeee47388d0/aten/src/ATen/native/native_functions.yaml#L9095), `sparse_grad` would be in `kwargs`. Additionally, since it seems to have no effect in the `gather` converter below, it can be removed/ignored, or a validator can be used to ensure it is `False`.